frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -68,26 +80,43 @@ const Review = ({ review }) => {
         className={classes.cardHeader}
       />
       <CardContent className={classes.cardContent}>
-        {/* <Typography className={classes.cardFont}>
-          <ShowMoreText
-            lines={1}
-            more={<ExpandMore />}
-            less={<ExpandLess />}
-            onClick={onClick}
-            expanded={expand}
-            width={0}
-            truncatedEndingComponent={"..."}
-            > */}
-        <Markdown children={review.comments} />
-        {/* </ShowMoreText> */}
-        {/* </Typography> */}
+        {/* 
+            217 is the approximate max number of chracters that can be visible before the need for
+            a scroll. As long as a review is longer than that, show "...read more"
+        */}
+        {review.comments.length > 220 ? (
```

Comment:
but what if there is a newline character?

Reply:
Remember that if we need the code to handle 2 situations then we need the code to have those situations built in.

Sometimes the review is long. Sometimes the reeview has newline characters. Sometimes both of these things.

Reply:
This handles new Lines for us: `<Typography noWrap>`